lib/resources/tasks/watch.ts

```diff
@@ -0,0 +1,100 @@
+import * as gulp from 'gulp';
+import * as minimatch from "minimatch";
+import * as gulpWatch from "gulp-watch";
+import * as debounce from "debounce";
+import { build } from 'aurelia-cli';
+import * as project from '../aurelia.json';
+import transpile from './transpile';
+import processMarkup from './process-markup';
+import processCSS from './process-css';
+import copyFiles from './copy-files';
+
+const debounceWaitTime = 100;
+let isBuilding = false;
+let pendingRefreshPaths = [];
+let watches = {};
+let watchCallback = () => { };
```

Comment:
I don't think so, because it really is the watch callback (it's just initialized to a noop).